src/browser/data/technologies.json

```diff
@@ -1,105 +1,150 @@
 {
   "0":[
     {
-      "id" : 8,
-      "name" : "scala",
+      "id" : 1,
+      "name" : "Scala",
       "src" : "assets/img/backend/scala.png"
     },
     {
-      "id" : 9,
+      "id" : 2,
       "name" : "akka",
       "src" : "assets/img/backend/akka.png"
     },
     {
-      "id" : 10,
+      "id" : 3,
       "name" : "play",
       "src" : "assets/img/backend/play.png"
     },
     {
-      "id" : 11,
+      "id" : 4,
       "name" : "slick",
       "src" : "assets/img/backend/slick.png"
     },
     {
-      "id" : 12,
+      "id" : 5,
       "name" : "spray",
       "src" : "assets/img/backend/spray.png"
+    },
+    {
+      "id" : 6,
+      "name" : "git",
+      "src" : "assets/img/generic/git.png"
     }
   ],
   "1" : [
     {
+      "id" : 10,
+      "name" : "Java Script",
+      "src" : "assets/img/frontend/js.png"
+    },
+    {
+      "id" : 11,
+      "name" : "HTML5",
+      "src" : "assets/img/frontend/html5.png"
+    },
+    {
+      "id" : 12,
+      "name" : "CSS3",
+      "src" : "assets/img/frontend/css3.png"
+    },
+    {
       "id" : 13,
-      "name" : "clojure",
-      "src" : "assets/img/clojure/clojure.png"
+      "name" : "Ember.js",
+      "src" : "assets/img/frontend/ember.png"
     },
     {
       "id" : 14,
-      "name" : "clojure-script",
-      "src" : "assets/img/clojure/clojurescript.png"
+      "name" : "Angular",
+      "src" : "assets/img/frontend/angular.png"
     },
     {
       "id" : 15,
-      "name" : "leiningen",
-      "src" : "assets/img/clojure/leiningen.png"
+      "name" : "Node.js",
+      "src" : "assets/img/frontend/node.png"
+    },
+    {
+      "id" : 16,
+      "name" : "React",
+      "src" : "assets/img/frontend/react.png"
+    },
+    {
+      "id" : 17,
+      "name" : "git",
+      "src" : "assets/img/generic/git.png"
     }
   ],
   "2" : [
     {
-      "id" : 16,
-      "name" : "swift",
-      "src" : "assets/img/mobile/swift.png"
+      "id" : 20,
+      "name" : "Android SDK",
+      "src" : "assets/img/android/android.png"
     },
     {
-      "id" : 17,
-      "name" : "objective-c",
-      "src" : "assets/img/mobile/objective-c.png"
+      "id" : 21,
+      "name" : "Java",
+      "src" : "assets/img/android/java.png"
     },
     {
-      "id" : 18,
-      "name" : "xcode",
-      "src" : "assets/img/mobile/xcode.png"
+      "id" : 22,
+      "name" : "Gradle",
+      "src" : "assets/img/android/gradle.png"
```

Comment:
I got those skills list from people whose are working with it and it was longer but we decided to keep just most important.